mantidproject/mantid
# IndirectDataReduction - EnergyTransfer Post Plot and Save

The plotting and saving of the results workspace has been changed to after the execution of the algorithm for the Indirect Energy Transfer tab.

**To test:**

Runs: 26176, 26174, 26173
\olympic\babylon5\Public\Louise McCann\Energy transfer
Run algorithm and check plot results as expected.
Also check all save formats are saved appropriately. 

Fixes #17255.

Release notes will be updated once post plot and save has been completed for all Indirect tabs.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
